go.mod

```
@@ -1,6 +1,6 @@
 module github.com/openshift/certman-operator
 
-go 1.21
+go 1.23.6
```

```suggestion
go 1.23.8
```
Can you check if we can have version 1.23.8 instead?